src/zrc/api/tests/test_validation.py

```diff
@@ -18,6 +18,7 @@
 )
 
 
+@override_settings(ZDS_CLIENT_CLASS='vng_api_common.mocks.MockClient')
```

Comment:
I think I'd prefer seeing the method `notify` be mocked away instead of replacing the entire client for all tests, this feels 'too big' :thinking:

Reply:
removed decorator